kinto/plugins/default_bucket/__init__.py

```diff
@@ -29,7 +28,9 @@ def create_bucket(request, bucket_id):
 
     # Fake context to instantiate a Bucket resource.
     context = RouteFactory(request)
-    context.get_permission_object_id = lambda r, i: '/buckets/%s' % bucket_id
+    context.get_permission_object_id = (lambda r,
+                                        i: Request.route_path('bucket-record',
+                                                              id=bucket_id))
```

Comment:
`route_path()` is a method. It means it has to be called on an instance of the class. Not the class itself.

In this case:
```python
bucket_uri = request.route_path('bucket-record', id=bucket_id)
context.get_permission_object_id = lambda r, i: bucket_uri
```